src/main/java/de/azapps/kafkabackup/sink/BackupSinkTask.java

```diff
@@ -196,7 +196,7 @@ public void flush() {
         // Note: we must iterate over all assigned partitions here,
         // since time-based rotation might kick in for partitions not present in last call to put().
         for (TopicPartition tp : partitionWriters.keySet()) {
-            log.info("Flushing {}", tp);
+            log.debug("Flushing {}", tp);
```

Comment:
Can we put a single "Flushing" info message before the for-loop? So we know it happens in general (but not for every single partition).